libs/wingsdk/src/std/string.ts

```diff
@@ -150,6 +150,24 @@ export class String {
     throw new Error("Abstract");
   }
 
+  /**
+   * Replaces occurrences of a substring within a string.
+   *
+   * @macro $self$.replace($args$)
+   *
+   * @param searchString The substring to search for.
+   * @param replaceString The replacement substring.
+   * @returns The modified string after replacement.
+   */
+  public stringReplace(
```

Comment:
shouldn't it be called just `replace`?

Reply:
Yeah, clearly an oversight.

Reply:
thanks! 

now just those two: 

> * add tests to this file: examples/tests/sdk_tests/std/string.w, testing both replacement of a string and a regex 
> * running pnpm build in your own machine, so it'll generate the snapshots and docs :)

Reply:
Do you know why it fails the build?

Reply:
I'm not sure Wing supports regex yet (Does it @Chriscbr?) 
if not, let's open an issue and add a comment to this method
maybe the easier thing to do is to accept only a string and put it into `new RegExp(...);` (maybe check if the string is  a regex first, otherwise, expressions like `string.replace("^", "any")` won't work quite well...)

Reply:
In fact, I noticed that there's an open issue to add `Regex` type to std library: #1894

I therefore proceeded to create another issue to add the feature to `string.replace()` in the future: #3644

Reply:
In fact, I noticed that there's an open issue to add `Regex` type to std library: #1894

I therefore proceeded to create another issue to add the feature to `String.replace()` in the future: #3644